tests/test_garbage_collect.py

```diff
@@ -23,23 +20,14 @@ def mock_logger():
         yield mocked_info
 
 
-def _get_time_created(d=0, h=0, s=0):
-    return (
-        datetime.datetime.utcnow() - datetime.timedelta(days=d, hours=h, seconds=s)
-    ).strftime("%Y-%m-%dT%H:%M:%SZ")
+def _get_created(d=0, h=0, s=0):
```

Comment:
Minor: I don't think you need to be naming these things with a leading `_` (up to you though).

Reply:
It's only to identify helper functions. Fine with changing this convention.